public/app/features/auth-config/AuthConfigPage.tsx

```diff
@@ -115,6 +115,9 @@ export const AuthConfigPageUnconnected = ({ providerStatuses, isLoading, loadSet
                 authType={provider.protocol}
                 enabled={providerStatuses[provider.id]?.enabled}
                 configPath={provider.configPath}
+                onClick={() => {
```

Comment:
This change is not affecting any tests. Shouldn't we have some tests for this page?

Reply:
hmm, it's a valid question. this interaction is coupled with sending a rudderstack event "only". not sure how to make a local test for testing that rudderstack would be picked up

Reply:
I believe this could benefit from testing only the onClick event, you don't need a rudderstack event

It's just an idea